pages/_lang/generate.vue

```diff
@@ -995,7 +993,7 @@ export default class extends Vue {
 
   public onClickRemoveScreenshot(
     e: Event,
-    screenshot: generator.Screenshot,
```

Comment:
Screenshot doesn't exist - it's the same type as Icon in the W3C spec. So we're reflecting that here, fixing a bunch of compiler errors.